packages/client/hmi-client/src/page/Home.vue

```diff
@@ -287,7 +298,7 @@ section {
 }
 
 .papers {
-	background: linear-gradient(180deg, var(--chevron-hover), #d5e8e5);
+	background: linear-gradient(180deg, #8bd4af1a, #d5e8e5);
```

Comment:
There is not CSS properties in `_variables.scss` for those colours? Are you going to reuse those colours anywhere?